test/views/RefundToTaxPayerViewSpec.scala

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import config.FrontendAppConfig
+import implicits.ImplicitDateFormatter
+import models.repaymentHistory.{RepaymentHistory, RepaymentHistoryModel, RepaymentItem, RepaymentSupplementItem}
+import org.jsoup.select.Elements
+import play.api.test.FakeRequest
+import testUtils.ViewSpec
+import views.html.RefundToTaxPayer
+
+import java.time.LocalDate
+
+class RefundToTaxPayerViewSpec extends ViewSpec with ImplicitDateFormatter {
+
+  lazy val mockAppConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
+
+  lazy val refundToTaxPayerView: RefundToTaxPayer = app.injector.instanceOf[RefundToTaxPayer]
+
+  object RefundToTaxPayerMessages {
+    val heading: String = messages("refund-to-taxpayer.heading")
+    val title: String = messages("titlePattern.serviceName.govUk", heading)
+    val agentTitle: String = messages("agent.titlePattern.serviceName.govUk", heading)
+    val titleWhenAgentView: String = messages("agent.titlePattern.serviceName.govUk", heading)
+
+    val tableHeadEstimatedDate: String = messages("refund-to-taxpayer.tableHead.estimated-date")
+    val tableHeadMethod: String = messages("refund-to-taxpayer.tableHead.method")
+    val tableHeadTotalRefund: String = messages("refund-to-taxpayer.tableHead.total-refund")
+    val tableHeadFurtherDetails: String = messages("refund-to-taxpayer.tableHead.further-details")
+    val tableHeadRequestedOn: String = messages("refund-to-taxpayer.tableHead.requested-on")
+    val tableHeadRefundReference: String = messages("refund-to-taxpayer.tableHead.refund-reference")
+    val tableHeadRequestedAmount: String = messages("refund-to-taxpayer.tableHead.requested-amount")
+    val tableHeadRefundAmount: String = messages("refund-to-taxpayer.tableHead.refund-amount")
+    val tableHeadInterest: String = messages("refund-to-taxpayer.tableHead.interest")
+    val tableHeadTotalInterest: String = messages("refund-to-taxpayer.tableHead.total-interest")
+    val variedInterest: String = s"${messages("refund-to-taxpayer.tableHead.total-interest")} ${messages("refund-to-taxpayer.tableHead.varied-interest-rates-value", "1.76", "2.01", "31 July 2021", "30 November 2021")}"
+
+    // todo check if it's needed
```

Comment:
Please update to remove the commented lines

Reply:
Thanks, Done